tests/TestingAppHost1/TestingAppHost1.AppHost/Program.cs

```diff
@@ -6,10 +6,14 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
 builder.AddRedis("redis1");
+var keycloak = builder.AddKeycloak("keycloak", 8080)
```

Comment:
You probably don't need to specify a port in this case.

Reply:
Dropped that change completely.